src/actors/build/cx_actor.jl

```diff
@@ -961,7 +965,7 @@ function optimize_outline(
             if hfs_thickness != lfs_thickness
                 hbuf = hfs_thickness - target_clearance
                 lbuf = lfs_thickness - target_clearance
-                r_obstruction, z_obstruction = buffer(r_obstruction, z_obstruction, hbuf, lbuf)
+                r_obstruction, z_obstruction = buffer(r_obstruction, z_obstruction, hbuf, hbuf)
```

Comment:
The goal here is to have the up/down clearance be equal to the hfs gap, instead of the average of the hfs and lfs gaps. This way works for me, but I'm sure there's a better way to do it...

Reply:
Before my fix:
![Screen Shot 2024-11-20 at 11 05 28 AM](https://github.com/user-attachments/assets/c2528cc4-f0c0-40aa-bda6-8a1edccec1af)


After my fix:
![Screen Shot 2024-11-20 at 11 02 18 AM](https://github.com/user-attachments/assets/24d3f249-e331-465f-adc6-e04c2ce3b973)